helm-misc.el

```diff
@@ -20,6 +20,7 @@
 (require 'helm)
 (require 'helm-help)
 (require 'helm-types)
+(require 'helm-ring)
```

Comment:
I guess you require helm-ring for `helm-kill-ring-max-offset`, I would prefer a new var, something like `helm-comint-max-offset`, this would allow separate values for both (kill-ring and comint) and also avoid requiring helm-ring just for that.
Thanks.